### PR TITLE
Hayden branch

### PR DESCRIPTION
**Pull request by haydenreich**
Complete beautiful experience

### CODE REVIEW

File: hatServer/controllers.py [hunk 1]
```diff
@@ -53,23 +67,32 @@ def create_survey():
...
 
-class Groups(Document):
-    group_name = StringField(required=True)
-    members = ListField(ReferenceField(Students))
+@app.route('/')
+@app.route('/about')
+def basic_pages(**kwargs):
+    return make_response(open('hatServer/templates/index.html').read())
 
+@app.route('/submit')
```
Comment (jepetty) [inline hatServer/controllers.py]: Will we ever hit this function/is it necessary?

File: hatServer/sortingHat/sortingHat.py [hunk 2]
```diff
@@ -90,7 +100,44 @@ def registerUser(student, groups):
                 known_score += 1 
             if attr in student.learn_skills and learn_score < MAX_SKILL_LEN:
                 learn_score += 1
-        groups[index].preferences[student] = calcGroupPreference(known_score, learn_score)
+        if student.ip_pref == 'NO_PREF':
```
Comment (jepetty) [inline hatServer/sortingHat/sortingHat.py:106]: Should I change storing of these variables on the angular side (i.e. change it from "noPref" to "NO_PREF")?

**Review by jepetty — APPROVED** [review]

File: hatServer/controllers.py (hunk 1 above)
Comment (haydenreich) [reply hatServer/controllers.py]: Nope, not at all necessary

File: hatServer/sortingHat/sortingHat.py (hunk 2 above)
Comment (haydenreich) [reply hatServer/sortingHat/sortingHat.py:106]: Commented in your PR, can change these on either end

File: hatServer/static/js/controllers/main.js [hunk 3]
```diff
@@ -8,6 +8,32 @@ sortingApp.controller('mainController', function($scope, userInformation) {
   }
 });
 
+sortingApp.controller('buildController', function ($scope, $http) {
+    data = {};
+
+    $scope.buildBitches = function () {
```
Comment (haydenreich) [inline hatServer/static/js/controllers/main.js:14]: Hahaha at least we're consistent

**Pull request merged**